file.go

```diff
@@ -129,12 +135,12 @@ func (f *File) Create(ctx Context) error {
 		return os.Mkdir(path, f.mode())
 	}
 
-	if f.Content != nil {
-		err := safeWriteContent(ctx, path, f.Content, f.MD5)
+	if content != nil {
+		err := safeWriteContent(ctx, path, content, f.MD5)
 		if err != nil {
 			return err
 		}
-	} else {
+	} else if f.Content == nil {
```

Comment:
Why not just doing an `else` without any condition ? Just taking into account the `content`variable:
```suggestion
	} else {
```
What should happen if `content == nil && f.Content != nil`? Does this mean that the file already exists?

Reply:
> Why not just doing an `else` without any condition ? Just taking into account the `content`variable:

We still need this method to create empty files. On these cases this method will be always called with `content == nil`.

> What should happen if `content == nil && f.Content != nil`? Does this mean that the file already exists?

Exactly.

Maybe this would be more readable if I split the methods to create and to write the files. :thinking: Wdyt?

Reply:
LGTM with these new methods ! thanks!